pixeltable/exec/expr_eval/expr_eval_node.py

```diff
@@ -285,7 +285,7 @@ async def __aiter__(self) -> AsyncIterator[DataRowBatch]:
                     if isinstance(self.error, excs.ExprEvalError):
                         raise self.error from self.error.exc
                     else:
-                        raise self.error
+                        raise excs.Error('An error occurred during expression evaluation.') from self.error
```

Comment:
This breaks the concurrency PR that's in review (it won't break the tests, but it breaks the behavior).

Reply:
Take a look at the expr_eval_node.py changes in that PR. We need to be able to see DBAPIErrors.

Reply:
I backed it out. But why would this prevent us from seeing DBAPIErrors?